scripts/process/process.storm_sites.R

```diff
@@ -57,20 +46,43 @@ process.all_sites <- function(viz = as.viz('all-sites')){
                          onclick=ifelse(is.featured, sprintf("openNWIS('%s', evt);", sites.sp@data$site_no), ""), 
                          stringsAsFactors = FALSE)
   
-  out<- list(sites.sp = sites.sp, mobile_featured = mobile_featured)
-  
-  saveRDS(out, viz[['location']])
+  saveRDS(sites.sp, viz[['location']])
 }
 
-process.getMobileSites <- function(viz = as.viz('mobile-sites')){
+#has a site passed flood stage or forecasted to?
+process.select_flood_sites <- function(viz = as.viz('storm-sites-flood')) {
+  library(dplyr)
+  depends <- readDepends(viz)
   
-  saveRDS(readDepends(viz)[['all-sites']]$mobile_featured, viz[['location']])
-}
-
-process.getDesktopSites <- function(viz = as.viz('storm-sites')){
```

Comment:
👍  thought these seemed a bit superfluous

Reply:
yeah... @jread-usgs was hoping to have a more strict filter for mobile. We never did it.